deepchecks/vision/checks/methodology/simple_feature_contribution.py

```diff
@@ -145,6 +150,15 @@ def compute(self, context: Context) -> CheckResult:
         df_train = pd.DataFrame(self._train_properties)
         df_test = pd.DataFrame(self._test_properties)
 
+        # PPS task type is inferred from label dtype. For computer vision tasks, it's safe to assume that unless
```

Comment:
don't we have task type explicitly instead of checking column type?

Reply:
As said in Slack, I have an issue with this assumption. Age prediction for example is a regression problem with and int label. 
Joining Matan's question